config-templates/shopify/customer.yaml

```diff
@@ -0,0 +1,27 @@
+data_source:
+  name: shopify_customers
+  owners:
+    - support@transformdata.io
+  sql_table: SHOPIFY.CUSTOMER ### Update to your opportunities schema.table
+  mutability:
```

Comment:
should we have a more realistic mutability example here?
```
  mutability:
    type: full_mutation
    type_params:
      update_cron: 0 2 * * *
```

Reply:
and in all other configs

Reply:
Agreed, I don't think these tables are going to be immutable.